source/isaaclab/isaaclab/controllers/differential_ik.py

```diff
@@ -146,7 +146,8 @@ def set_command(
                     self.ee_pos_des, self.ee_quat_des = apply_delta_pose(ee_pos, ee_quat, self._command)
                 else:
                     self.ee_pos_des = self._command[:, 0:3]
-                    self.ee_quat_des = self._command[:, 3:7]
+                    if self.cfg.command_type == "velocity":
```

Comment:
```suggestion
                    if self.cfg.command_type != "velocity":
```
? or do i misunderstand

Reply:
This else is only applicable for velocity/linear-velocity control

Reply:
But we're in a branch here that can never be velocity? Line 122 is only pose or position